ingest/src/test/java/com/connexta/ingest/IngestApplicationIntegrationTest.java

```diff
@@ -97,8 +97,6 @@ public void testSuccessfulIngestRequest() throws Exception {
         .expect(requestTo(endpointUrlTransform))
         .andExpect(method(HttpMethod.POST))
         .andExpect(header("Accept-Version", endpointsTransformVersion))
-        .andExpect(jsonPath("$.bytes").value(TEST_FILE_SIZE))
-        .andExpect(jsonPath("$.mimeType").value(TEST_MIME_TYPE))
```

Comment:
Why did this PR change the tests for the Transform request when the PR only bumps the version for the Ingest API?